### PR TITLE
Fixed drawing outside of DrawingView control

### LINKED ISSUE #1679: [Bug] [DrawingView] [Android] draw outside of control
### Description

When drawing something on the DrawingView Control and moving out of the control, the lines are still drawn

### Steps to Reproduce

1. Open Android Sample App
2. Open DrawingView Sample
3. Draw a line starting from inside the DrawingView
4. Draw the line outside of the control

### Expected Behavior
Lines outside of the control should not be visible/drawn

### Actual Behavior
Lines can be drawn outside of the control

### Basic Information

- Version with issue: 1.3.0-pre2
- IDE: VS 2019 16.11.2
- Platform Target Frameworks: 
  - Android: 11.0

### Reproduction imagery
image

### PR DESCRIPTION
**Pull request by circa94**
### Description of Bug ###
Fixed the bug on Android where it was possible to draw outside of the DrawingView control.
It's not possible anymore to draw outside of the control:
image

### Issues Fixed ###

- Fixes  #1679 

### PR Checklist ###

- [ ] Has a linked Issue, and the Issue has been `approved`
- [ ] Has tests (if omitted, state reason in description)
- [ ] Has samples (if omitted, state reason in description)
- [ ] Rebased on top of main at time of PR
- [ ] Changes adhere to coding standard

- [ ] Updated [documentation](https://github.com/MicrosoftDocs/xamarin-communitytoolkit)

### CODE REVIEW

**Comment by brminnick** [comment]
Thanks! Can you add Unit Tests and a reproduction to the Sample App?

File: samples/XCT.Sample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj [hunk 1]
```diff
@@ -32,6 +32,12 @@
     <DefineConstants>DEBUG;</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <AotAssemblies>false</AotAssemblies>
```
Comment (VladislavAntonyuk) [inline samples/XCT.Sample.Android/Xamarin.CommunityToolkit.Sample.Android.csproj:35]: fixes Run in Debug mode

File: src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs [hunk 2]
```diff
@@ -168,7 +158,7 @@ public override bool OnInterceptTouchEvent(MotionEvent ev)
 			return base.OnInterceptTouchEvent(ev);
 		}
 
-		IList<Point> NormalizePoints(IList<Point> points)
+		IList<Point> NormalizePoints(IEnumerable<Point> points)
```
Comment (pictos) [inline src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs:161]: Just out of curiosity, why did you change the parameter type?

Comment (VladislavAntonyuk) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs:161]: I firstly added IList in the previous commit. we can use a more abstract type here (we do not need Add,Remove behavior from IList)

Comment (pictos) [reply src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.android.cs:161]: thanks for the answer.

**Review by VladislavAntonyuk — APPROVED** [review]
